vcs/git/repo.go

```diff
@@ -16,6 +16,7 @@ import (
 	"golang.org/x/tools/godoc/vfs"
 	"sourcegraph.com/sourcegraph/go-vcs/vcs"
 	"sourcegraph.com/sourcegraph/go-vcs/vcs/gitcmd"
+	"sourcegraph.com/sourcegraph/go-vcs/vcs/internal"
```

Comment:
I saw it used in another project somewhere (I can't remember which one exactly) and just thought it looked a lot cleaner :)